chunky/src/java/se/llbit/chunky/block/Head.java

```diff
@@ -55,7 +55,7 @@ public Entity toEntity(Vector3 position) {
   }
 
   @Override
-  public boolean isBlockEntity() {
+  public boolean isBlockEntity(CompoundTag entityTag) {
     return type == Kind.PLAYER;
```

Comment:
Re previous comment: This logic could be moved to `getBlockEntity` and return null if the type isn't a player. Same for `WallHead`